REQUIRE

```diff
@@ -1,4 +1,3 @@
 julia 0.7
-BinDeps 0.8.10
-@!windows CMakeWrapper
-@osx Homebrew 0.7.0
+BinaryProvider 0.3.0
+CMake
```

Comment:
better to drop `REQUIRE` and add `Project.toml` instead.

Reply:
should be in another PR

Reply:
Let's start CMake with its latest released version (1.1.1, it looks like).